liquigraph-core/src/main/java/org/liquigraph/core/configuration/ConnectionConfigurationByUri.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.core.configuration;
+
+import com.google.common.base.Optional;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import static com.google.common.base.Throwables.propagate;
+
+public class ConnectionConfigurationByUri implements ConnectionConfiguration {
+
+    private final String uri;
+    private final Optional<String> username;
+    private final Optional<String> password;
+
+    public ConnectionConfigurationByUri(String uri,
+                                        Optional<String> username,
+                                        Optional<String> password) {
+        this.uri = uri;
+        this.username = username;
+        this.password = password;
+    }
+
+    @Override
+    public Connection get() {
+        try {
+            Class.forName("org.neo4j.jdbc.Driver");
```

Comment:
This can (and should) be left out in the `master_3x` branch. Maybe it's superfluous in the `master` as well?